spatial_cell.cpp

```diff
@@ -698,6 +698,18 @@ namespace spatial_cell {
             displacements.push_back((uint8_t*) &(this->parameters[CellParams::RHOQ_DT2]) - (uint8_t*) this);
             block_lengths.push_back(sizeof(Real));
          }
+
+         // send RHOQE, uses duplicate of RHOQ transfer flag
```

Comment:
Why not use RHOQ above and add * 2 to the size? And add a comment in common.h that they should be consecutive. I find that less error-prone (with the comment in common.h) than duplicating these guys here.

Reply:
Fine